examples/CantileverDistributedLoad/cantilever_distrubuted_load_postprecessing.py

```diff
@@ -0,0 +1,396 @@
+    # sanity check of mass second moment of inertia
+    if (mass_second_moment_of_inertia < Tolerance.atol()).all():
+        message = "Mass moment of inertia matrix smaller than tolerance, please check provided radius, density and length."
+        log.warning(message)
+
+    # Inverse of second moment of inertia
+    inv_mass_second_moment_of_inertia = np.zeros(
+        (MaxDimension.value(), MaxDimension.value(), n_elements)
+    )
+    for i in range(n_elements):
+        # Check rank of mass moment of inertia matrix to see if it is invertible
+        assert (
+            np.linalg.matrix_rank(mass_second_moment_of_inertia[..., i])
+            == MaxDimension.value()
+        )
+        inv_mass_second_moment_of_inertia[..., i] = np.linalg.inv(
+            mass_second_moment_of_inertia[..., i]
+        )
+
+    # Shear/Stretch matrix
+    if not shear_modulus:
+        log.info(
+            """Shear modulus is not explicitly given.\n
+            In such case, we compute shear_modulus assuming poisson's ratio of 0.5"""
+        )
+        shear_modulus = youngs_modulus / (2.0 * (1.0 + 0.5))
+
+    # Value taken based on best correlation for Poisson ratio = 0.5, from
+    # "On Timoshenko's correction for shear in vibrating beams" by Kaneko, 1975
+    alpha_c = 27.0 / 28.0
+    shear_matrix = np.zeros(
+        (MaxDimension.value(), MaxDimension.value(), n_elements), np.float64
+    )
+    for i in range(n_elements):
+        np.fill_diagonal(
+            shear_matrix[..., i],
+            [
+                alpha_c * shear_modulus * A0[i],
+                alpha_c * shear_modulus * A0[i],
+                youngs_modulus * A0[i],
+            ],
+        )
+
+    # Bend/Twist matrix
+    bend_matrix = np.zeros(
+        (MaxDimension.value(), MaxDimension.value(), n_voronoi_elements + 1), np.float64
+    )
+    for i in range(n_elements):
+        np.fill_diagonal(
+            bend_matrix[..., i],
+            [
+                youngs_modulus * I0_1[i],
+                youngs_modulus * I0_2[i],
+                shear_modulus * I0_3[i],
+            ],
+        )
+    if ring_rod_flag:  # wrap around the value in the last element
+        bend_matrix[..., -1] = bend_matrix[..., 0]
+    for i in range(0, MaxDimension.value()):
+        assert np.all(
+            bend_matrix[i, i, :] > Tolerance.atol()
+        ), " Bend matrix has to be greater than 0."
+
+    # Compute bend matrix in Voronoi Domain
+    rest_lengths_temp_for_voronoi = (
+        np.hstack((rest_lengths, rest_lengths[0])) if ring_rod_flag else rest_lengths
+    )
+    bend_matrix = (
+        bend_matrix[..., 1:] * rest_lengths_temp_for_voronoi[1:]
+        + bend_matrix[..., :-1] * rest_lengths_temp_for_voronoi[0:-1]
+    ) / (rest_lengths_temp_for_voronoi[1:] + rest_lengths_temp_for_voronoi[:-1])
+
+    return (
+        mass_second_moment_of_inertia,
+        inv_mass_second_moment_of_inertia,
+        bend_matrix,
+    )
+
+
+def plot_video_with_surface(
```

Comment:
This function is repeated in all of these examples. Consider only have one of these?